modules/aws_nodegroup/aws-nodegroup.json

```diff
@@ -33,9 +33,14 @@
       "default": "t3.medium",
       "description": "The [ec2 instance type](https://aws.amazon.com/ec2/instance-types/) for the nodes."
     },
+    "ami_type": {
+      "type": "boolean",
+      "description": "The AMI type to use for the nodes.\nFor more information about this, please visit [here](https://docs.aws.amazon.com/eks/latest/APIReference/API_Nodegroup.html#AmazonEKS-Type-Nodegroup-amiType)\nNote: Currently, \"CUSTOM\" ami type is not supported.\n",
+      "default": "AL2_x86_64"
+    },
     "use_gpu": {
       "type": "boolean",
-      "description": "Should we expect and use the gpus present in the ec2?",
+      "description": "Should we expect and use the gpus present in the ec2?\nNote: This input would be deprecated in the coming releases. Please switch to using `ami_type`\n",
```

Comment:
Shoudl we say, which value they should use in ami_type? `AL2_x86_64_GPU` right

Reply:
Yeah, the `ami_type` is correct, but not sure whether it should be mentioned in the Documentation. We are keeping this currently support for now, but when we remove this from the usage completely, then we can add this.